Khan/perseus
# Show images on their own line when included in a radio widget.

## Summary:

When images are added to a radio widget, they are often intended to be on their own line. Creators indicate this by entering two new-line (`\n\n`) characters. This gets rendered as a separate `div.paragraph` in the HTML. This code change assigns CSS to the elements that match this rendered pattern. Content that has only a single new-line character around the image are rendered with the same `div.paragraph`, and therefore do not receive this styling.

Issue: LEMS-1917

## Test plan:
1. Launch Storybook locally (`yarn start`)
1. Navigate to "[Perseus > Widgets > Radio > Select With Images](http://localhost:6006/?path=/story/perseus-widgets-radio--select-with-images)"
1. Note the options that have images on their own line. _(There should be only 1 that is not on its own line.)_

## Affected behavior:

### Before
![Images in Radio Widget - Before](https://github.com/Khan/perseus/assets/13896410/01ccdb07-db97-463b-bf61-392e4da48b92)


### After
![Images in Radio Widget - After](https://github.com/Khan/perseus/assets/13896410/717f3b5c-b678-462f-ab3b-134310fbdd97)
